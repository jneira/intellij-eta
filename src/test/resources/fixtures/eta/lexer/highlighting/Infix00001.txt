ITmodule ('module')
WHITE_SPACE (' ')
ITconid ('Infix00001')
WHITE_SPACE (' ')
ITwhere ('where')
WHITE_SPACE ('\n\n')
ITvarid ('val')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITinteger ('1')
WHITE_SPACE (' ')
ITvarid ('`Prelude.elem`')
WHITE_SPACE (' ')
ITobrack ('[')
ITinteger ('1')
ITcomma (',')
WHITE_SPACE (' ')
ITinteger ('2')
ITcomma (',')
WHITE_SPACE (' ')
ITinteger ('3')
ITcbrack (']')
WHITE_SPACE ('\n')
