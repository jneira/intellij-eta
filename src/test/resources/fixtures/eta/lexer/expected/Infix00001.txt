ITmodule ('module')
WHITE_SPACE ('')
ITconid (' Infix0000')
WHITE_SPACE ('1')
ITwhere (' wher')
WHITE_SPACE ('e\n\n')
ITvocurly ('')
WHITE_SPACE ('')
ITvarid ('val')
WHITE_SPACE (' ')
ITequal ('=')
WHITE_SPACE (' ')
ITinteger ('1')
WHITE_SPACE (' ')
ITbackquote ('`')
WHITE_SPACE ('')
ITqvarid ('Prelude.elem')
WHITE_SPACE ('')
ITbackquote ('`')
WHITE_SPACE (' ')
ITobrack ('[')
WHITE_SPACE ('')
ITinteger ('1')
WHITE_SPACE ('')
ITcomma (',')
WHITE_SPACE (' ')
ITinteger ('2')
WHITE_SPACE ('')
ITcomma (',')
WHITE_SPACE (' ')
ITinteger ('3')
WHITE_SPACE ('')
ITcbrack (']')
WHITE_SPACE ('\n')
ITsemi ('')